nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/main/java/org/nuxeo/elasticsearch/shield/ShieldInitializationService.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * (C) Copyright 2014-2016 Nuxeo SA (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.nuxeo.elasticsearch.shield;
+
+import org.elasticsearch.client.transport.TransportClient;
+import org.elasticsearch.common.settings.Settings;
+import org.elasticsearch.shield.ShieldPlugin;
+import org.nuxeo.elasticsearch.api.ESClientInitializationService;
+import org.nuxeo.elasticsearch.config.ElasticSearchRemoteConfig;
+import org.nuxeo.runtime.api.Framework;
+
+/**
+ * @since 9.1
+ */
+public class ShieldInitializationService implements ESClientInitializationService {
+
+    private static final String ES_SHIELD_ADMIN_USERNAME = "elasticsearch.shield.admin.username";
+
+    private static final String ES_SHIELD_ADMIN_PASSWORD = "elasticsearch.shield.admin.password";
+
+    @Override
+    public Settings initializeSettings(ElasticSearchRemoteConfig config) {
+
+        String shieldAdminName = Framework.getProperty(ES_SHIELD_ADMIN_USERNAME, "Administrator");
+        String shieldAdminPasswd = Framework.getProperty(ES_SHIELD_ADMIN_PASSWORD, "Administrator");
+
+        Settings.Builder builder = Settings.settingsBuilder()
+                .put("cluster.name", config.getClusterName())
+                .put("client.transport.nodes_sampler_interval", config.getSamplerInterval())
+                .put("client.transport.ping_timeout", config.getPingTimeout())
+                .put("client.transport.ignore_cluster_name", config.isIgnoreClusterName())
+                .put("client.transport.sniff", config.isClusterSniff())
+                .put("shield.user", shieldAdminName + ":" + shieldAdminPasswd);
+
```

Comment:
This repeats most of the code from `ESClientInitializationServiceImpl`, please avoid the duplication. You could have a protected method that returns the base `Builder` constructed.